src/core/device.c

```diff
@@ -709,16 +709,34 @@ static void device_update_found_by_name(Manager *m, const char *path, DeviceFoun
 }
 
 static bool device_is_ready(sd_device *dev) {
+        int r;
+
         assert(dev);
 
-        if (device_is_renaming(dev) > 0)
+        r = device_is_renaming(dev);
+        if (r < 0)
+                log_device_error_errno(dev, r, "Failed to check if device is renaming: %m");
```

Comment:
Maybe, better to mention that it is assumed that the device is not renaming.

Reply:
Also, please downgrade this to debug.

Reply:
Why should this be a debug log? Unless we expect this to happen often, it seems to make more sense to log at error level so it's more noticeable when it happens

Reply:
Downgraded the error logging in this function to warning instead

Reply:
I thought this is a kind of library function. But OK, Let's see what happens.